src/west/_bootstrap/main.py

```diff
@@ -115,26 +115,59 @@ def init(argv):
     This exits the program with a nonzero exit code if fatal errors occur.'''
     init_parser = argparse.ArgumentParser(
         prog='west init',
-        description='Bootstrap initialize a Zephyr installation')
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=
+'''
+Initializes a West instance.
+
+In more detail, does the following:
+
+  1. Clones the manifest repository to west/manifest, and the west repository
+     to west/west
+
+  2. Creates a marker file west/{}
+
+  3. Creates an initial configuration file west/config
+
+'west init' can be rerun on an already initialized West instance to update
+configuration settings. This is just an alternative to manually editing
+west/config. Only explicitly passed configuration values (e.g.
+--mr NEW_MANIFEST_REVISION) are updated.
+
+Note: If you update the west/manifest revision or URL, the change will take
+effect whenever the configuration setting is next read, e.g. in `west fetch`.
+'west init' only updates configuration settings.
+'''.format(WEST_MARKER))
+
     init_parser.add_argument(
         '-b', '--base-url',
-        help='''Base URL for both 'manifest' and 'zephyr' repositories; cannot
```

Comment:
Can we "fix" this so that multiline strings are used throughout? Seems cleaner (and is consistent with the main west).

Reply:
Seems a bit overkill. It's consistent with the other strings, just breaking it across lines.

Reply:
Or do you mean using a single-line multiline string for short descriptions that fit on a single line? What'd be the point of that? Just adds a bunch of line noise, IMO.

Reply:
> Seems a bit overkill. It's consistent with the other strings, just breaking it across lines.

No, it's not consistent with the general pattern, which is to use a multiline string whenever the help string has to occupy multiple lines. Is this needed because of the change to a RawDescriptionHelpFormatter? The argparse internals should be reflowing the help normally otherwise.



> Or do you mean using a single-line multiline string for short descriptions that fit on a single line? What'd be the point of that? Just adds a bunch of line noise, IMO.

I mean using a multiline string for long descriptions that span multiple lines.

Reply:
Phew, thought you meant all of them.

Changed.